### PR TITLE
Add/front/content/edtion

### LINKED ISSUE #21: Front - Member - Upload file
User Story:

    As a member I want to have access to a form in order to upload a document to submit

### PR DESCRIPTION
**Pull request by LemouD**
Closes #21 #33 

## Proposed Changes

  - Add Front New Content 
  - Add Front Content Interface

### CODE REVIEW

File: public/css/Style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.content_index{
+    font-size: 14px ;
+}
```
Comment (AitZak) [inline public/css/Style.css:3]: add empty line at the end of file as it's recommended

File: templates/content/edit.html.twig [hunk 2]
```diff
@@ -7,7 +7,10 @@
 
     {{ include('content/_form.html.twig', {'button_label': 'Update'}) }}
 
-    <a href="{{ path('content_index') }}">back to list</a>
+    <button type="submit" class="btn btn-outline-success">
+        <a href="{{ path('content_index') }}"> <span class="btn-label"><i class="fas fa-sign-in-alt"></i></span>
+            back to list</a></button>
+
```
Comment (AitZak) [inline templates/content/edit.html.twig]: Remove empty line

**Review by AitZak — COMMENTED** [review]
Sounds good to me.
Just watch out unnecessary empty line

**Review by LemouD — COMMENTED** [review]
Looks good for me

**Review by AitZak — APPROVED** [review]

**Pull request merged**